torch/testing/_internal/common_methods_invocations.py

```diff
@@ -588,22 +588,48 @@ def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad, **kw
 
     return inputs
 
+def sample_inputs_addmm_non_fusible_nodes(op_info, device, dtype, requires_grad, **kwargs):
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
+    tests_list = [
+        ((S, M), (S, S), (S, M)),
+    ]
+    tests_require_resizing = [
+        ((1,), (S, S), (S, M)),
+        ((), (S, S), (S, M))
+    ]
+    test_cases = [*tests_list] if for_inplace_variant else [*tests_list, *tests_require_resizing]  # type: ignore
+    inputs = tuple(SampleInput(make_tensor(shape_a, device, dtype, requires_grad=requires_grad),
+                               args=(make_tensor(shape_b, device, dtype,
+                                                 requires_grad=requires_grad),
+                                     make_tensor(shape_c, device, dtype,
+                                                 requires_grad=requires_grad)))
+                   for shape_a, shape_b, shape_c in test_cases)
+    return inputs
+
 def sample_inputs_addmm(op_info, device, dtype, requires_grad, **kwargs):
-    input = SampleInput(
-        make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=requires_grad),
-        args=(
-            make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=requires_grad),
-            make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=False)))
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
```

Comment:
You don't need to do this for this PR (though you could), but you could call `sample_inputs_addmm_non_fusible_nodes` and iterate over the samples to add the kwargs.